docs/en_US/Compressor/Pruner.md

```diff
@@ -133,12 +133,15 @@ The above configuration means that there are 5 times of iterative pruning. As th
 * **sparsity:** The final sparsity when the compression is done.
```

Comment:
would be better to add an index at the top of this file below title.

Reply:
resolved